### ISSUE TITLE
BibLaTeX styles use invalid options

### ISSUE THREAD

**Issue by LaTeXer** [borisveytsman/acmart#504]
When using the `acmart` BibLaTeX styles, the build fails due to invalid options.

MWE:
```
\documentclass{article}
% or \documentclass[sigconf,natbib=false]{acmart}

\usepackage[style=acmnumeric]{biblatex}

\begin{document}
\end{document}
```

fails with

```
…/texmf-dist/tex/latex/acmart/acmnumeric.bbx:885: Package xkeyv
al Error: `halid' undefined in families `blx@opt@pre'.

See the xkeyval package documentation for explanation.
```

The same is true for `style=acmauthoryear`. Copying the style file to the local build directory and removing the offending line triggers similar errors for `swhid`, `swlabels`, `vcs`, and `license`. Once all these are gone, everything seems to work fine.

**Comment by SECtim** [comment]
I had the same problem, until it just went away - I investigated and found that these options are part of the [`biblatex-software` package](https://ctan.org/pkg/biblatex-software), which seems to be relatively recent.

In particular, my TeXLive distribution (2019) did not contain that package; after upgrading to TeXLive 2021, these options no longer lead to errors (if you're on Ubuntu >= 22.04 (jammy) or similar, look for the `texlive-bibtex-extra` Ubuntu package; note: prior LTS versions of that package do NOT contain `biblatex-software`).

In other words, instead of messing with the `.bbx` file, do one of the following:
- Update your TeX distribution
- Install `biblatex-software` manually
- Put `software.bbx` from the `biblatex-software` package next to your `.tex` file(s)
